### PR TITLE
Fix #82: Document configuration for Azure upload.

### LINKED ISSUE #82: Add Azure File Storage example to uploader config documentation
The uploader config documentation should be extended by an example of an configuration that uses an Azure File Storage.

### PR DESCRIPTION
**Pull request by salsolatragus**
Fixes #82.

Added documentation of the option to upload traces to an Azure File Storage.

I added the documentation to the repository, because the `userguide.md` file was more recently updated than the repo wiki. However, from the main `README.md` we currently link to the wiki, which is not in sync with `userguide.md`. What is the preferred documentation strategy here?

### CODE REVIEW

**Comment by karottenreibe** [comment]
@salsolatragus the wiki is outdated and should not be used. We should remove all the content from there and link to the documentation in the repo instead

Unfortunately, I won't have time in the near future to review your changes :-( it's probably best if you find someone else to review them. e.g. @zipperle

**Comment by salsolatragus** [comment]
@karottenreibe I've replaced all links from the code to the wiki by links to the in-code documentation. Cannot deactivate the wiki, as this requires admin privileges.

I'm happy with anyone's review :)

**Comment by salsolatragus** [comment]
@zipperle @mpdeimos @karottenreibe this is a really small change (only docu update), would be awesome to have it closed soon 0:-)

File: documentation/userguide.md [hunk 1]
```diff
@@ -307,6 +307,25 @@ match:
       fileUpload: http://localserver.localdomain:8080
 ```
 
+## Example: Azure File Storage
+
+**UploadDaemon.yaml:**
+
+  To upload traces to an Azure File Storage first obtain the [connection string](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string) for your storage account.
```
Comment (karottenreibe) [inline documentation/userguide.md]: please don't use inline links. Use named references and put the link at the bottom of the file. That makes reading the markdown easier while editing

**Comment by karottenreibe** [comment]
great work!

**Comment by karottenreibe** [comment]
i asked Martin to remove the Wiki since I don't have the necessary permissions either ;-)

**Pull request merged**